xml/hardening.xml

```diff
@@ -3445,6 +3445,325 @@ Account Expires: Never</screen>
    </para>
   </sect1>
   <xi:include href="hardening_pam_stack.xml"/>
+  <sect1 xml:id="sec-sec-prot-restrict-root">
+   <title>Restricting &rootuser; Logins</title>
+
+   <para>
+    By default, the &rootuser; user is assigned a password
+    and can login via various methods like on any local terminal, in a
+    graphical session, or remotely via SSH. These methods should be restricted
```

Comment:
We have both `local console` and `local terminal` used throughout the document currently, but not `system console` or `system terminal`. So I'd like to avoid introducing another term for the same thing.

Reply:
I will push a change without the `system console` bit.